pom.xml

```diff
@@ -140,7 +140,10 @@
         <dependency>
             <groupId>org.testobject</groupId>
             <artifactId>testobject-java-api</artifactId>
-            <version>0.0.26</version>
+            <version>0.0.27</version>
+            <!-- Using local jar for development. Remove when releasing -->
```

Comment:
You can also just install the java-api locally with mvn install. You don't need to have a configuration like this